config/prow/config.yaml

```diff
@@ -226,3 +226,40 @@ presets:
   env:
     - name: GOPPC64
       value: power9
+# knative powervs preset
+- labels:
+    preset-knative-powervs: "true"
+  env:
+    - name: TF_VAR_powervs_api_key
+      valueFrom:
+        secretKeyRef:
+          name: knative-apikey
+          key: apikey
+    - name: KO_FLAGS
+      value: --platform=linux/ppc64le
+    - name: PLATFORM
+      value: linux/ppc64le
+    - name: INGRESS_CLASS
+      value: contour.ingress.networking.knative.dev
+    - name: KO_DOCKER_REPO
+      value: icr.io/upstream-k8s-registry/knative
+    - name: DOCKER_CONFIG
+      value: /root/.docker
+
+  volumeMounts:
+    - name: ssh-key-volume
+      mountPath: /root/.ssh/
+      readOnly: true
+    - name: config-json-volume
+      mountPath: /root/.docker/
+      readOnly: true
+
+  volumes:
+    - name: ssh-key-volume
+      secret:
+        secretName: ssh-key-secret
+        defaultMode: 384 
+    - name: config-json-volume
```

Comment:
Cosmetic suggestion, you may remove the suffix `-volume` to names.

Reply:
Removed the `--volume` suffix from the secrets